internal/hosts/metadata.go

```diff
@@ -2,6 +2,7 @@ package hosts
 
 type Metadata struct {
 	Cluster     string `mapstructure:"ha-cluster,omitempty"`
+	ClusterId   string `mapstructure:"ha-cluster-id,omitempty"`
```

Comment:
Now that we store the `ClusterId`, I don't know if we need the `Cluster` entry (the name). This is just used to filter the consul catalog. Maybe for the future if we add some filters by name?

Reply:
For now I'm using it to display the name (while we use the ID on the link for the href). This is a bit lazy and we could just pull that information, but since it was there already I thought it might be simpler for now to keep it. WDYT?

Reply:
Ok, it is fine